packages/lwt-windows.2.5.2/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "1.2"
+maintainer: "Pierre Boutillier <pierre.boutillier@laposte.net>"
+authors: [
+  "Jérôme Vouillon"
+  "Jérémie Dimino"
+]
+homepage: "http://github.com/ocsigen/lwt"
+dev-repo: "git://github.com/ocsigen/lwt"
+bug-reports: "http://github.com/ocsigen/lwt/issues"
+license: "LGPL with OpenSSL linking exception"
+
+build: [
+  ["ocaml" "setup.ml" "-configure"
+    "--override" "os_type" "Win32"
+    "--override" "flexlink" "%{lib}%/flexdll-windows/flexlink"
+    "--override" "ext_dll" ".dll"
+    "--override" "datarootdir" "%{prefix}%/windows-sysroot/share"
+    "--disable-pthread"
```

Comment:
the _oasis file of lwt then:
```
Flag pthread
  Description: Use pthread
  Default$: !os_type(Win32)
```
I'm sorry but I won't try to enable-pthread myself.
(which does not mean I'm mad, I'm extremely thankful for your patience and all your advices)